pom.xml

```diff
@@ -58,10 +58,11 @@
     <!-- We are ignoring java doc warnings - this is because we are using JDK 11. Ideally we should not do that. -->
     <doclint>none</doclint>
     <sonar.analyzer.commons>2.4.0.1317</sonar.analyzer.commons>
-    <sonar.version>9.14.0.375</sonar.version>
+    <sonar.version>9.15.0.435</sonar.version>
     <sonar.api.impl.version>10.0.0.68432</sonar.api.impl.version>
     <jdk.min.version>11</jdk.min.version>
     <maven.compiler.release>${jdk.min.version}</maven.compiler.release>
+    <slf4j.version>2.0.7</slf4j.version>
```

Comment:
My (very limited) understanding is that:
- the explicit version was not needed before because a default was provided for it in : https://github.com/SonarSource/sonar-plugin-api/pull/77/files#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30deL16;
- that should be true, except that the version is now provided via `version('slf4j', '1.7.30')`;
- why it doesn't work, I don't l know;
- why it only fails on some tests, I don't know either.

In any case, I have now realized that this version is defined in the top-level pom, and not in the plugin pom.
So, yes, it is in test scope and is a safe dependency.